apack.json

```diff
@@ -4,8 +4,9 @@
   "description": "Graphical configuration tool for application and libraries based on Zigbee Cluster Library.",
   "path": [".", "node_modules/.bin/", "ZAP.app/Contents/MacOS"],
   "requiredFeatureLevel": "apack.core:9",
-  "featureLevel": 106,
+  "featureLevel": 107,
   "uc.triggerExtension": "zap",
+  "uc.sdkProvidedProperties": "zcl.matterZclJsonFile,zcl.matterTemplateJsonFile,zcl.zigbeeZclJsonFile,zcl.zigbeeTemplateJsonFile",
```

Comment:
Do not see zcl.zigbeeZclJsonFile,zcl.zigbeeTemplateJsonFile in the GSDK. How are these populated?

Reply:
yes please